tests/constraints/test_list_constraints.py

```diff
@@ -91,45 +66,39 @@ def test_handle_constrained_list_with_max_items(
 def test_handle_constrained_list_with_min_items(
     min_items: int,
 ) -> None:
-    field = create_model_field(str, min_items=min_items)
-    assert issubclass(field.annotation, ConstrainedList)
     result = handle_constrained_collection(
         collection_type=list,
         factory=ModelFactory,
-        field_meta=field,
+        field_meta=FieldMeta.from_type(List[str], name="test", random=Random()),
         item_type=str,
-        max_items=field.annotation.max_items,
-        min_items=field.annotation.min_items,
+        min_items=min_items,
     )
     assert len(result) >= min_items
 
 
-def test_handle_constrained_list_with_different_types() -> None:
-    for t_type in ModelFactory.get_provider_map():
-        field = create_model_field(t_type, min_items=1)
-        assert issubclass(field.annotation, ConstrainedList)
-        result = handle_constrained_collection(
-            collection_type=list,
-            factory=ModelFactory,
-            field_meta=field.children[0] if field.children else field,
-            item_type=t_type,
-            max_items=field.annotation.max_items,
-            min_items=field.annotation.min_items,
-        )
-        assert len(result) > 0
+@pytest.mark.skipif(sys.version_info < (3, 9) and pydantic_version == 2, reason="fails on python 3.8 with pydantic v2")
```

Comment:
Issue with pydantic v2?

Reply:
on python v3.8 only. they seem to still need to hammer out some stuff there.